packages/mobx-state-tree/src/types/utility-types/custom.ts

```diff
@@ -13,8 +13,8 @@ import {
 export interface CustomTypeOptions<S, T> {
     /** Friendly name */
     name: string
-    /** given a serialized value, how to turn it into the target type */
-    fromSnapshot(snapshot: S): T
+    /** given a serialized value and environment, how to turn it into the target type */
+    fromSnapshot(snapshot: S, env: any): T
```

Comment:
Should be `env?: any` I think, since this is a public interface. Beyond that LGTM